lib/mapbox-gl-geocoder.css

```diff
@@ -232,4 +232,11 @@
     margin-right: -5px;
   }
 
+  .mapbox-gl-geocoder--error{
+    color: darkgray;
+    padding: 8px;
```

Comment:
Consider padding that is consistent with existing styles (`6px 12px`):

![Screen Shot 2019-05-01 at 10 52 04 AM](https://user-images.githubusercontent.com/108094/57032423-30bc7380-6bff-11e9-81bf-2981671a56f7.png)